### DISCUSSION TITLE
Good evening, I have an issue with the script, I'll provide it below.

### DISCUSSION THREAD

**Discussion by Empereuphenix** [community/community#123449] [Pull Requests]
### Select Topic Area

Bug

### Body

Traceback (most recent call last):

ModuleNotFoundError: No module named 'pyaes'
317 INFO: PyInstaller: 6.6.0, contrib hooks: 2024.6
318 INFO: Python: 3.12.3
342 INFO: Platform: Windows-11-10.0.22624-SP0
Script file 'loader-o.py' does not exist.

**Comment by IgorD-lab** [comment]
did you install pyaes?

$ pip install pyaes

**Reply by Empereuphenix** [reply]
Yes

**Reply by IgorD-lab** [reply]
I see that error is mentioning that 'loader-o.py' does not exist meaning that it cant find it. Try double checkign the path. Not much else I can see from the error message.

**Reply by Empereuphenix** [reply]
It's only that, so it's anything but precise.

**Comment by lseaborn1** [comment]
i've had the EXACT same error, someone please help!

**Comment by ilay05** [comment]
Yea I got this too, help please !
